### PR TITLE
web-apps(front): Update the README

### PR DESCRIPTION
**Pull request by kimwnasptd**
Update the readme with detailed commands on how to consume the library as well as developer guidelines.

/cc @StefanoFioravanzo 
/cc @thesuperzapper 
/assign @kimwnasptd

### CODE REVIEW

File: components/crud-web-apps/common/frontend/kubeflow-common-lib/README.md [hunk 1]
```diff
@@ -1,27 +1,43 @@
...
+* Reduce the development effort required to propagate changes to all the web apps
+* Minimize the code duplication between our Kubeflow web apps
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 8.3.20, which is required to build and run the unit tests.
+
+## Local development
+In order to use this library while developing locally  an Angular app you will need to:
+1. Build the `kubeflow` node module from this source code
+2. Link the produced module to your global npm modules
+3. Link the `kubeflow` module in the npm modules of you app
```
Comment (yanniszark) [inline components/crud-web-apps/common/frontend/kubeflow-common-lib/README.md:14]: nit: your

**Review by yanniszark — CHANGES_REQUESTED** [review]
Thanks @kimwnasptd, overall lgtm.
Do you want to fix the one typo before merging?

File: components/crud-web-apps/common/frontend/kubeflow-common-lib/README.md (hunk 1 above)
Comment (kimwnasptd) [reply components/crud-web-apps/common/frontend/kubeflow-common-lib/README.md:14]: Good catch, thanks!
Pushed a new commit that fixes if

**Pull request merged**